frontend/src/pages/modelRegistry/screens/RegisterModel/useRegisterModelData.ts

```diff
@@ -19,6 +19,13 @@ export type RegistrationCommonFormData = {
   modelLocationURI: string;
   versionCustomProperties?: ModelRegistryCustomProperties;
   modelCustomProperties?: ModelRegistryCustomProperties;
+  modelSourceGroup?: string;
+  modelSourceId?: string;
+  modelSourceName?: string;
+  catalogSourceName?: string;
+  catalogRepositoryName?: string;
+  catalogModelName?: string;
+  catalogModelTag?: string;
```

Comment:
Since we aren't mapping these to editable form fields and they're just being passed along with the form data similar to the `versionCustomProperties` and `modelCustomProperties`, I think we can simply add one property here for the bundle of them. In fact it doesn't even need to be unique to the model source, it's just additional properties for us to slap on the artifact - so you could add:

```ts
additionalArtifactProperties?: Partial<ModelArtifact>;
```